### PR TITLE
Added django-rest-hooks recipe

### PR DESCRIPTION
**Pull request by Korijn**
Not sure if I got `build.script` and `source.url` right.

### CODE REVIEW

File: recipes/django-rest-hooks/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  name: django-rest-hooks
+  version: {{ version }}
+
+source:
+  fn: django-rest-hooks-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/d1/7e/234d5f6e67e5f3d2ca64e3c142b31d58742b02fb552cdc9410eeb167538c/django-rest-hooks-{{ version }}.tar.gz
+  sha256: c121ffbc1ee4a116bd9d5b39e3c97152f409d84dd4d13e121465d20a03ac0dcf
+
+build:
+  script: python setup.py install
```
Comment (ivoflipse) [inline recipes/django-rest-hooks/meta.yaml]: `python setup.py install --single-version-externally-managed --record=record.txt`

Comment (Korijn) [inline recipes/django-rest-hooks/meta.yaml]: I get an error when I include that:

```
> python setup.py install --single-version-externally-managed --record=record.txt
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help
error: option --single-version-externally-managed not recognized
Command failed: C:\WINDOWS\system32\cmd.exe /c call bld.bat
```

File: recipes/django-rest-hooks/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{% set version = "1.2.2" %}
+
+package:
+  name: django-rest-hooks
+  version: {{ version }}
+
+source:
+  fn: django-rest-hooks-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/d1/7e/234d5f6e67e5f3d2ca64e3c142b31d58742b02fb552cdc9410eeb167538c/django-rest-hooks-{{ version }}.tar.gz
```
Comment (ivoflipse) [inline recipes/django-rest-hooks/meta.yaml]: `https://pypi.io/packages/source/d/django-rest-hooks/django-rest-hooks-{{ version }}.tar.gz`

File: recipes/django-rest-hooks/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  name: django-rest-hooks
+  version: {{ version }}
+
+source:
+  fn: django-rest-hooks-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/django-rest-hooks/django-rest-hooks-{{ version }}.tar.gz
+  sha256: c121ffbc1ee4a116bd9d5b39e3c97152f409d84dd4d13e121465d20a03ac0dcf
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (ocefpaf) [inline recipes/django-rest-hooks/meta.yaml]: It seems that `django-rest-hooks` does not use `setuptools`. See https://github.com/zapier/django-rest-hooks/blob/master/setup.py#L1

You will need to drop the `--single-version-externally-managed --record=record.txt` here.

File: recipes/django-rest-hooks/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    - django >=1.4
+    - requests
+
+test:
+  imports:
+    - rest_hooks
+
+about:
+  home: http://github.com/zapier/django-rest-hooks
+  license: BSD License
```
Comment (ocefpaf) [inline recipes/django-rest-hooks/meta.yaml]: I tried to figure out which BSD (2-Clause or 3-Clause) they use, but the [LICENSE.md](https://github.com/zapier/django-rest-hooks/blob/master/LICENSE.md) file did not help. Can you open an issue upstream asking the correct version of the license?

Comment (Korijn) [inline recipes/django-rest-hooks/meta.yaml]: Done: https://github.com/zapier/django-rest-hooks/issues/30

Comment (ocefpaf) [inline recipes/django-rest-hooks/meta.yaml]: We can merge this before they answer BTW as this is "correct" according to the repository. But we will need to fix the feedstock once they answer. So it is up to you if you want to wait or merge.

Comment (Korijn) [inline recipes/django-rest-hooks/meta.yaml]: Let's merge. I'll open a PR once they reply.

Comment (Korijn) [inline recipes/django-rest-hooks/meta.yaml]: See https://github.com/conda-forge/django-rest-hooks-feedstock/pull/1

**Pull request merged**